partner_helper/tests/test_split_address.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+from openerp.tests.common import SingleTransactionCase
```

Comment:
SingleTransactionCase is rarely use. Here no problem because only 1 test.
But generally tests are executed in an independant way.